src/Vault/Api/AuthApi.cs

```diff
@@ -2563,6 +2563,12 @@ public interface IAuthSync : IApiAccessor
         /// </summary>
         /// <exception cref="VaultApiException">Thrown when fails to make API call</exception>
         /// <param name="jwtMountPath">Path that the backend was mounted at</param>
+
+        /// <param name="clientNonce"> (optional)</param>
+
+        /// <param name="code"> (optional)</param>
+
+        /// <param name="state"> (optional)</param>
```

Comment:
It seems some extra blank lines got introduced in the comments as a result of the template changes.

Reply:
That wasn't me :-) - I didn't touch that part of the template - but it was easy enough to fix up anyway.

Reply:
I'm going to revert the fix, as the PR check is spuriously failing on it!

As the extra blank lines are not introduced by anything I changed in this PR - they're an existing defect in the template - can we merge as originally submitted, and worry about why the PR check is spuriously failing separately?

Reply:
This white spaces are likely leftover from the switch to handlebar templates. Unfortunately, the formatting we apply doesn't handle whitespaces.

Reply:
OK, after much confusion, I finally figured it out.

I'd naively removed `{{unless required}}` because why would you omit query parameters that were required?!

But it turns out `{{unless required}}` is being used as a dubious heuristic to omit the `"list"` query parameter.

And to put another layer on it, my `dotnet format` locally was .NET 6, whilst the GitHub action is using .NET Core 3.1 ... and I'm guessing the newer .NET is smart enough to automatically delete `<param name="list">` blocks where there is no parameter named `list` in that function, whilst the older one doesn't.

So, my mistake in the template was being auto-fixed by my formatter - but not by GitHub actions' formatter!

Anyway, all sorted now, should be good to review and merge.

Reply:
Thanks for sorting this out!

Reply:
> And to put another layer on it, my dotnet format locally was .NET 6, whilst the GitHub action is using .NET Core 3.1 ... and I'm guessing the newer .NET is smart enough to automatically delete <param name="list"> blocks where there is no parameter named list in that function, whilst the older one doesn't.

We should probably upgrade to the latest version of .NET in GHA.